numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4908,5 +4908,6 @@ PyMODINIT_FUNC PyInit__multiarray_umath(void) {
         PyErr_SetString(PyExc_RuntimeError,
                         "cannot load multiarray module.");
     }
+    Py_XDECREF(m);
```

Comment:
`m` may be uninitialized if `npy_cpu_init` fails. Maybe just put the module creation first?